inspirehep/utils/citations.py

```diff
@@ -42,8 +44,7 @@ def citations(self):
 
         out = []
         row = []
-        recid = self.record['control_number']
-        es_query = Query('refersto:' + str(recid)).search()
+        es_query = Query('refersto:' + str(self.recid)).search()
```

Comment:
@jmartinm I notice here we are deliberately limiting the number of citation to the first 9999 ones. Since anyway we are truncating the results, maybe we can simply gave up and provide the first 200 (or 500)? In this way the page is not unnecessarily bloated and one can always open a search page.